src/lib/parquet/src/Flow/Parquet/Writer/ColumnChunkBuilders.php

```diff
@@ -103,6 +104,10 @@ public function uncompressedSize() : int
 
     private static function createFlatColumnBuilder(FlatColumn $column, Options $options, Compressions $compressions) : ColumnChunkBuilder
     {
+        if ($column->type() === PhysicalType::INT32 || $column->type() === PhysicalType::INT64) {
```

Comment:
since Delta Binary Packed encoding was added in Parquet V2, we should use it only when V2 is set (or when it's requested by the user, but that requires #696 )